src/test/create-jwt-token.ts

```diff
@@ -1,12 +1,12 @@
-import Jwt from "jsonwebtoken";
-import {TokenData} from "../schema";
+import Jwt from 'jsonwebtoken';
+import { TokenData } from '../api/schema';
 
-export function createJwtToken(options: {payload: TokenData; extendedExpiration?: boolean}): string {
-  const {payload, extendedExpiration} = options;
+export function createJwtToken(options: { payload: TokenData; extendedExpiration?: boolean }): string {
+  const { payload, extendedExpiration } = options;
 
-  const expiresIn = extendedExpiration ? "7d" : "1d";
+  const expiresIn = extendedExpiration ? '7d' : '1d';
 
-  const token = Jwt.sign(payload, process.env.JWT_TOKEN as string, {expiresIn});
+  const token = Jwt.sign(payload, process.env.JWT_TOKEN as string, { expiresIn });
```

Comment:
nit: pode ser isso tb
```suggestion
  const token = Jwt.sign(payload, process.env.JWT_TOKEN!, { expiresIn });
```